pyiron_base/job/core.py

```diff
@@ -739,18 +739,33 @@ def copy_to(self, project, new_job_name=None, new_database_entry=True, copy_file
 
         Args:
             project (JobCore/ProjectHDFio/Project): project to copy the job to
-            new_database_entry (bool): [True/False] to create a new database entry - default True
+            input_only (bool): [True/False] Whether to copy only the input. (Default is False.)
+            new_database_entry (bool): [True/False] Whether to create a new database entry. If input_only is True then
```

Comment:
I think I would rather raise an error if these are conflicting. I'm open to push-back on that though.

Reply:
Since this is just shuffling existing behavior I'd prefer a warning.  As far as grep can tell `input_only=True` only in protocols (protocol.primitive.one_state:254) though, so maybe we do a hard switch in a separate PR and fix it in contrib before the next version bump.

Reply:
Sure thing, a warning is completely sufficient for now. Actually probably much safer than suddenly making it an error, as per second point:

Yeah, I wasn't aware of heavy usage in the code base, but since it's public I guess we also need to think about it being used in notebooks we know nothing about. (And thus for a transition period a warning is important!)